### PR TITLE
Update image-spec and runtime-spec to v1.0.0

### PR DESCRIPTION
**Pull request by nalind**
Update to just-released versions of image-spec and runtime-spec, and the latest version of runtime-tools.  The runtime-spec dropped the `Platform` field, so stop trying to set it when calling out to a runtime as part of handling a `run` command or `RUN` instruction.

### CODE REVIEW

File: run.go [hunk 1]
```diff
@@ -153,12 +153,6 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	}()
 	g := generate.New()
 
-	if b.OS() != "" {
```
Comment (TomSweeneyRedHat) [inline run.go]: Just curious, was this changed for the spec update or just something you meant to change?

Comment (nalind) [inline run.go]: The field was dropped from the runtime spec, and the runtime-tools API no longer provides the methods we were calling here.

Comment (TomSweeneyRedHat) [inline run.go]: LGTM and thx for the 411!

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ 16eb3c6

**Pull request closed without merging**